tasks/12.txt

```diff
@@ -5,7 +5,7 @@ Write a SQL query to get Product list of twenty most expensive products:
 Order products by price.
 
 db.products.aggregate([
-   {$project:{"ProductName":1,"UnitPrice":1}},
-   {$sort:{"UnitPrice":-1}},
-   {$limit:20}
+{$limit:20},
+{$sort:{"UnitPrice":-1}},
+{$project:{"ProductName":1,"UnitPrice":1}},
 ])
```

Comment:
sort logic will be invalid after first limit

Reply:
Hmm,really. So,  you are attentive)